test/e2e/page-objects/pages/home/asset-list.ts

```diff
@@ -178,15 +178,11 @@ class AssetListPage {
   }
 
   async clickOnAsset(assetName: string): Promise<void> {
-    const buttons = await this.driver.findElements(this.tokenListItem);
-    for (const button of buttons) {
-      const text = await button.getText();
-      if (text.includes(assetName)) {
-        await button.click();
-        return;
-      }
-    }
-    throw new Error(`${assetName} button not found`);
+    const buttons = await this.driver.findElement({
+      css: this.tokenListItem,
+      text: assetName,
+    });
+    await buttons.click();
```

Comment:
better to use `await driver.clickElement({ css: this.tokenListItem, text: assetName,}); `to avoid race condition